packages/contentful--app-scripts/src/utils.ts

```diff
@@ -108,11 +108,17 @@ export function getEntityFromManifest<Type extends 'actions' | 'functions'>(type
       )}.`,
     );
 
+    const fieldMappingEvent = "graphql.field.mapping";
+    const queryEvent =  "graphql.query";
+
     const items = (manifest[type] as FunctionAppAction[] | ContentfulFunction[]).map((item) => {
       const allowNetworks = Array.isArray(item.allowNetworks)
         ? item.allowNetworks.map(stripProtocol)
         : [];
 
+      const accepts = 'accepts' in item ? Array.isArray(item.accepts) ? item.accepts : undefined : undefined;
+      const hasInvalidEvent = accepts?.find((event) => ![fieldMappingEvent, queryEvent].includes(event));
```

Comment:
Nit: I would move this close to the `if` statement. So its faster to get to the definition when you read the `if` statement. 

```suggestion
      const hasInvalidEvent = accepts?.some((event) => ![fieldMappingEvent, queryEvent].includes(event));
```

As you are not looking for the event itself.